src/components/MenuPanel/MenuPanel.js

```diff
@@ -347,36 +354,41 @@ class AnimatedMenuPanel extends React.Component {
                         0, 0
                       )
                     `
-                ),
-                opacity: progress.interpolate(v => Number(v > 0)),
-              }}
-            >
-              <Viewport>
-                {({ height }) => (
-                  <MenuPanel viewportHeight={height} {...props} />
-                )}
-              </Viewport>
-            </Wrap>
-          )}
-        </Spring>
-        <Viewport>
-          {({ below }) =>
-            below('medium') && (
-              <Overlay
-                opened={Boolean(openProgress)}
-                onClick={onCloseMenuPanel}
-              />
-            )
-          }
-        </Viewport>
-      </React.Fragment>
+                  ),
+                  opacity: progress.interpolate(v => Number(v > 0)),
+                }}
+              >
+                <Viewport>
+                  {({ height }) => (
+                    <MenuPanel viewportHeight={height} {...props} />
+                  )}
+                </Viewport>
+              </Wrap>
+              {autoClosing && (
+                <Overlay
+                  onClick={onCloseMenuPanel}
+                  style={{
+                    /* by leaving a 1px edge Android users can swipe to open
+                     * from the edge of their screen when an iframe app is being
+                     * used */
+                    width: progress.interpolate(p =>
+                      p === 0 ? '10px' : '100vw'
```

Comment:
Should we update the comment to 10px? Is the increase to make the swipe register more consistently?

Reply:
ooooh this was for debugging purposes, good catch :open_mouth: